tests/wakebox/timeout-exceeded/pass.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# It's not valid to call wakebox with an empty PATH.
+# So we fill PATH with some typical values.
+export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
+
+${1}/wakebox -p input.json -o result.json
+
+trap "rm result.json hello" EXIT
+
+[ "$(cat result.json | jq '."timed-out"')" = "true" ] && [ -f "hello" ] && exit 0
```

Comment:
This checks that fuse-waked wrote the file to disk, which it would have anyway
We care that the json itself is populated so that if any other tools read it they know about the file that was written

something like:
```suggestion
[ "$(cat result.json | jq '."timed-out"')" = "true" ] && exit 0
[ "$(cat result.json | jq -r .outputs[])" = "hello" ] && exit 0
```